test/integration/model/update.test.js

```diff
@@ -35,6 +35,53 @@ describe(Support.getTestDialectTeaser('Model'), () => {
         }));
     });
 
+    it('should not check for notNull Violation for undefined values', function() {
+      const ownerId = 2;
+      let accountRowId;
+      return this.Account.create({
+        ownerId,
+        name: Math.random().toString()
+      }).then(account => {
+        accountRowId = account.get('id');
+        const accountVal = {
+          name: Math.random().toString(),
+          ownerId: undefined
+        };
+        return this.Account.update(accountVal, {
+          where: {
+            id: accountRowId
+          }
+        });
+      }).then(() => {
+        return this.Account.findById(accountRowId);
+      }).then(account => {
+        expect(account.ownerId).to.be.equal(ownerId);
+      });
+    });
+
+    it('should ignore undefined values', function() {
```

Comment:
Looks like duplicate of test above, Please add a test here https://github.com/sequelize/sequelize/blob/master/test/integration/instance/update.test.js so we can be sure instance method update also ignores `undefined` values

Reply:
It was one for not null props and one for the nullable props (as it might behave different) but I guess the first one is enough